app/src/main/res/layout/admin_auth_activity.xml

```diff
@@ -40,91 +40,78 @@
         app:titleTextColor="@color/white" />
     </com.google.android.material.appbar.AppBarLayout>
 
-    <FrameLayout
-      android:layout_width="0dp"
-      android:layout_height="0dp"
-      app:layout_constraintBottom_toBottomOf="parent"
+    <View
+      android:id="@+id/admin_auth_toolbar_shadow_view"
+      android:layout_width="match_parent"
+      android:layout_height="6dp"
+      android:background="@drawable/toolbar_drop_shadow"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toBottomOf="@id/admin_auth_app_bar_layout">
+      app:layout_constraintTop_toBottomOf="@+id/admin_auth_app_bar_layout" />
 
-      <androidx.constraintlayout.widget.ConstraintLayout
-        android:layout_width="match_parent"
-        android:layout_height="match_parent">
-
-        <TextView
-          android:id="@+id/admin_auth_heading_textview"
-          android:layout_width="match_parent"
-          android:layout_height="wrap_content"
-          android:layout_marginStart="28dp"
-          android:layout_marginTop="40dp"
-          android:layout_marginEnd="28dp"
-          android:fontFamily="sans-serif-medium"
-          android:maxLines="2"
-          android:text="@string/admin_auth_heading"
-          android:textColor="@color/oppiaPrimaryText"
-          android:textSize="20sp"
-          app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toTopOf="parent" />
+    <TextView
+      android:id="@+id/admin_auth_heading_textview"
+      style="@style/Heading2"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="28dp"
+      android:layout_marginTop="12dp"
+      android:layout_marginEnd="28dp"
+      android:maxLines="2"
+      android:text="@string/admin_auth_heading"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@id/admin_auth_app_bar_layout" />
 
-        <TextView
-          android:id="@+id/admin_auth_sub_text"
-          android:layout_width="0dp"
-          android:layout_height="wrap_content"
-          android:layout_marginStart="28dp"
-          android:layout_marginTop="12dp"
-          android:layout_marginEnd="28dp"
-          android:fontFamily="sans-serif"
-          android:text="@string/admin_auth_sub"
-          android:textColor="@color/oppiaAdminAuthSecondaryText"
-          android:textSize="16sp"
-          app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toBottomOf="@id/admin_auth_heading_textview" />
+    <TextView
+      android:id="@+id/admin_auth_sub_text"
+      style="@style/Subtitle1"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
```

Comment:
You can remove `layout_width` and `layout_height` and introduce `horizontal_bias="0.0"`

Reply:
Done.